test/integration/user_can_create_account_test.rb

```diff
@@ -0,0 +1,64 @@
+require "test_helper"
+
+class UserCanCreateAccountTest < ActionDispatch::IntegrationTest
+  test "user can create account and sees profile" do
+    items = create_list(:item, 2)
+
+    visit items_path
+
+    first(:button, "Add to Cart").click
+    assert page.has_content?("You added #{items.first.title} to your cart.")
+
+    visit item_path(items.last)
+
+    click_button "Add to Cart"
+    assert page.has_content?("You added #{items.last.title} to your cart.")
+
+    find("#shopping_cart").click
+
+    assert_equal cart_path, current_path
+    items.each do |item|
+      assert page.has_content?(item.title)
+      assert page.has_content?(item.description)
+      assert page.has_content?(item.price)
+      assert page.has_css?("img[src*='#{item.image_path}']")
+    end
```

Comment:
Similarly, our other test already checked for all of the parts of the items. If that test passes, we know these lines are true. Maybe we can assert just the title of the item in this test.